.clang-format

```diff
@@ -1,17 +1,17 @@
 ---
 Language:        Cpp
 # BasedOnStyle:  LLVM
-AccessModifierOffset: -4
+AccessModifierOffset: -8
```

Comment:
This is definitely an improvement for editors that render tabs as four spaces.

I was thinking about turning on function argument binpacking for the same reason. Are you okay with that?